content/SCALE/SCALETutorials/Credentials/ManageLocalGroups.md

```diff
@@ -18,32 +18,59 @@ If the network uses a directory service, import the existing account information
 
-Enter a unique number for the group ID in **GID** that TrueNAS uses to identify a Unix group. Enter a number above 1000 for a group with user accounts or for a system service enter the default port number for the service as the GID.  Enter a name for the group. The group name cannot begin with a hyphen (-) or contain a space, tab, or any of these characters: colon (:), plus (+), ampersand (&), hash (#), percent (%), carat (^), open or close parentheses ( ), exclamation mark (!), at symbol (@), tilde (~), asterisk (*), question mark (?) greater or less than (<) (>), equal (=). You can only use the dollar sign ($) as the last character in a user name.
+Enter a unique number for the group ID in **GID** that TrueNAS uses to identify a Unix group.
+Enter a number above 3000 for a group with user accounts or enter the default port number as the GID for a system service.
 
-**Allowed sudo commands**, **Allow all sudo commands**, **Allowed sudo commands with no password** and **Allow all sudo commands with no password** allows the group members to act as the system administrator using the [sudo](https://www.sudo.ws/) command. Leave it disabled for better security.
+Enter a name for the group.
+The group name cannot begin with a hyphen (-) or contain a space, tab, or any of these characters: colon (:), plus (+), ampersand (&), hash (#), percent (%), carat (^), open or close parentheses ( ), exclamation mark (!), at symbol (@), tilde (~), asterisk (*), question mark (?) greater or less than (<) (>), equal (=).
+You can only use the dollar sign ($) as the last character in a group name.
+
+**Allowed sudo commands**, **Allow all sudo commands**, **Allowed sudo commands with no password** and **Allow all sudo commands with no password** allow members of the group to act as root using the [sudo](https://www.sudo.ws/) command.
+To allow sudo commands, use **Allowed sudo commands** or **Allowed sudo commands with no password** to list the executable path(s) to specific sudo commands allowed for group members.
+<file>/usr/bin/</file> is the default location for command executables.
+Alternately, click the checkbox for **Allow all sudo commands** or **Allow all sudo commands with no password**.
+
+Exercise caution when allowing sudo commands, especially without password prompts.
+It is recommended to limit this privilege to trusted users and specific commands to minimize security risks.
 
 To allow Samba permissions and authentication to use this group, select **Samba Authentication**.
 
 To allow more than one group to have the same group ID (not recommended), select **Allow Duplicate GIDs**.
+Use only if absolutely necessary, as duplicate GIDs can lead to unexpected behavior.
+
+## Managing Groups
+
+Click the <span class="material-icons">expand_more</span> arrow or anywhere on a row to expand that group and show the group management buttons.
```

Comment:
Since the behavior of clicking on a row allows the user to not have to click the arrow, maybe we should just tell them to click anywhere on a row and do away with the reference to the expand arrow. This is a change from Anglefish and CORE where you had to click on the expand arrows.